### ISSUE TITLE
Why did the value of "extent" changed when updating fix version?

### ISSUE THREAD

**Issue by dr0i** [hbz/lobid-resources#1894]
Results from https://github.com/hbz/lobid-resources/pull/1890.
Please check the cause of these changes https://github.com/hbz/lobid-resources/pull/1890/commits/d945a4cbd47fc4223fbe3ca3bb140806add6da73#diff-845841b3fac5568bdd0662f2a8e7c5e015e9ea3d4f776da1709510b50c2a9d1d.

**Comment by blackwinter** [comment]
That's due to the destructive behaviour of `copy_field` (see metafacture/metafacture-fix#313).

```perl
do list(path: "300  .b", "var": "$i")
  copy_field("$i", "@300b")
end

# ...

paste("extent", "@300a1", "@300b_punct", "@300c1_punct", "@300e_punct", "@300a2_punct", "@300c2_punct", join_char: "")
```

```xml
<datafield tag="300" ind1=" " ind2=" ">
  <subfield code="a">153 Seiten</subfield>
  <subfield code="b">Fotografien</subfield>
  <subfield code="b">Diagramme</subfield>
</datafield>
```

Previously, `@300b` contained both values (implicit array creation) and `paste` only used the first one. With the new version, the second value overwrites the first one instead.

According to the MARC standard, [`300$$b`](https://www.loc.gov/marc/bibliographic/bd300.html) is not repeatable, so both results may be considered equally valid.

**Comment by dr0i** [comment]
Thx for this explanation - I suspected something like this. It's especially good to know that this is a cataloging error.
